src/runtimeChoiceSelector.ts

```diff
@@ -0,0 +1,95 @@
+import * as _ from "lodash";
+import * as path from "path";
+import * as clc from "cli-color";
+import * as semver from "semver";
+
+import * as FirebaseError from "./error";
+import * as utils from "./utils";
+
+// have to require this because no @types/cjson available
+// tslint:disable-next-line
+var cjson = require("cjson");
+
+const MESSAGE_FRIENDLY_RUNTIMES: any = {
```

Comment:
You could make the types on these `{[key: string]: string}` for slightly more complete typing.

Reply:
Oh love that, was looking for that, thanks!